song-server/src/main/java/bio/overture/song/server/controller/AnalysisController.java

```diff
@@ -135,15 +137,16 @@ public void patchUpdateAnalysis(
   @PutMapping(value = "/publish/{id}")
   @SneakyThrows
   @PreAuthorize("@studySecurity.authorize(authentication, #studyId)")
-  public ResponseEntity<String> publishAnalysis(
-      @RequestHeader(value = AUTHORIZATION, required = false) final String accessToken,
+  public ResponseEntity<Object> publishAnalysis(
+      // @RequestHeader(value = AUTHORIZATION, required = false) final String accessToken,
```

Comment:
why is this accessToken header commented?

Reply:
commented the header for testing, reverted back